### ISSUE TITLE
The label "Does the data contain column headers" is not associated with the yes radio button: A11y_ML.NET Model Builder_Advanced data options_Screenreader.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3022]
### GitHubTags
#A11ySev3; #A11yTCS;#Model Builder - Machine Learning for .NET Developers;#DesktopApp; #Win32;#BM-MlnetModelBuilder-Win32-March25;#Info and Relationships; #Win32;#WCAG1.3.1;#Element:RadioButton;#A11yMAS; 

### Actual Result: 
The label "Does the data contain column headers" is not associated with the yes radio button but when the focus moves to the no radio button, the screen reader announces the label information.
### Expected Result:
The label "Does the data contain column headers" should be associated with the yes radio button and the screen reader should read it.
### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3
### Repro Steps:
**Prerequisites:**
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Right-click on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario and select a scenario.
5. Select training environment (Select local(CPU).
6. Tab till next step and press enter, "select file from the data source type and add a file".
7. Tab till advanced data options and press enter.
8. Tab till data formatting and press enter.
9. Verify that the label "Does the data contain column headers" is not associated with the yes radio button.

### User Impact:
Screen reader users will encounter an issue as they will get confused yes radio button is associated with which field.
Recommendations:

### Attachment:

https://github.com/user-attachments/assets/ebbeb0f0-3e51-49ed-b0f9-02ccfa0e7e04

**Comment by Mana-swi** [comment]
#a11ydb-group-not-exposed;
